plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -61,6 +64,45 @@ public override IEnumerable<FileModel> Prebuild(ImmutableList<FileModel> models,
                         collection.AddRange(result.Models);
                     }
                 }
+
+                var duplicatePathes = filePathesDict.Where(x => x.Value.Item1).Select(x => x.Key).ToList();
+                if (duplicatePathes.Any())
+                {
+                    foreach (var path in duplicatePathes)
```

Comment:
>foreach (var path in duplicatePathes) [](start = 20, length = 37)

`foreach (var path in filePathesDict.Where(x => x.Value.Item1).Select(x => x.Key))`